.changeset/fix_the_use_strict_directive_insertion_logic_for_shebang_and_top_leading_comments.md

```diff
@@ -2,21 +2,21 @@
 "@biomejs/biome": patch
 ---
 
-Fix [#4841](https://github.com/biomejs/biome/issues/4841), shebang and top leading comments in cjs files are now handled correctly
+Fixed [#4841](https://github.com/biomejs/biome/issues/4841): Shebang and top leading comments in `.cjs` files are now handled correctly
 
-- shebang only (keep it as is)
+## Example: shebang only (keep it as is)
```

Comment:
I'm not how headers will render in the final changlog. Keep an eye on it just in case